src/cascadia/TerminalSettingsModel/FileUtils.cpp

```diff
@@ -43,6 +44,33 @@ namespace winrt::Microsoft::Terminal::Settings::Model
         return baseSettingsPath;
     }
 
+    // Returns a path like C:\Users\<username>\AppData\Local\Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState
+    // to the path of the stable release settings
+    std::filesystem::path GetReleaseSettingsPath()
+    {
+        static std::filesystem::path baseSettingsPath = []() {
+            wil::unique_cotaskmem_string localAppDataFolder;
+            // KF_FLAG_FORCE_APP_DATA_REDIRECTION, when engaged, causes SHGet... to return
+            // the new AppModel paths (Packages/xxx/RoamingState, etc.) for standard path requests.
+            // Using this flag allows us to avoid Windows.Storage.ApplicationData completely.
+            THROW_IF_FAILED(SHGetKnownFolderPath(FOLDERID_LocalAppData, KF_FLAG_NO_PACKAGE_REDIRECTION, nullptr, &localAppDataFolder));
+
+            // Returns a path like C:\Users\<username>\AppData\Local
+            std::filesystem::path parentDirectoryForSettingsFile{ localAppDataFolder.get() };
+
+            //Appending \Packages\Microsoft.WindowsTerminal_8wekyb3d8bbwe\LocalState to the settings path
+            parentDirectoryForSettingsFile /= ReleaseSettingsFolder;
+
+            if (!IsPackaged())
+            {
+                parentDirectoryForSettingsFile /= UnpackagedSettingsFolderName;
+            }

```

Comment:
I'm not sure this is right for unpackaged versions of the Terminal. I might want to tap in @DHowett for this bit, hes a bit more familiar with running the Terminal unpackaged. 

Just looking at my own files, I've only got a `%localAppData%\Microsoft\Windows Terminal`.

Heck, I'm not sure this is ever relevant for Preview builds running unpackaged - I think they always just use `%localAppData%\Microsoft\Windows Terminal` anyways. I might be wrong here.